components/brave_wallet_ui/components/extension/post-confirmation/common/transaction_intent.tsx

```diff
@@ -126,11 +126,19 @@ export const TransactionIntent = (props: Props) => {
   const transactionConfirmed =
     transaction.txStatus === BraveWallet.TransactionStatus.Confirmed
 
+  // Currently we only get transaction.swapInfo.receiver info
+  // for lifi swaps. Core should also return this value
+  // for all other providers.
+  const swapOrBridgeRecipient =
+    transaction.swapInfo?.provider === 'lifi'
+      ? transaction.swapInfo?.receiver ?? ''
+      : transaction.fromAddress ?? ''
+
   const recipientLabel = getAddressLabel(
     isERC20Approval
       ? txApprovalTarget
       : isSwapOrBridge && transactionConfirmed
```

Comment:
Do you remember why we have `transactionConfirmed` in this condition?

Reply:
Actually looks like `isConfirmed` is already being handled bellow in the `secondDuringValue` declaration. Went ahead and removed this condition.